src/components/@commons-feature/calendar/Calendar.tsx

```diff
@@ -0,0 +1,102 @@
+import React from 'react';
+import { getMonthly } from 'apis/getSchedule';
+import { useQuery } from '@tanstack/react-query';
+import Text from 'components/@commons/Text';
+import { Badge, BadgeCont, DateCont, InnerDayBox, MonthBox, OutterDayBox, WeekGrid } from './CalendarStyle';
+import FlexContainer from '../../@commons/FlexContainer';
+import { NextButton, PrevButton } from '../../@commons/iconButtons';
+import { dateAtom } from 'components/admin-MainPage/AdminScheduleSection';
+import { useAtom } from 'jotai';
+
+interface Props {
+  selectedId: number;
+}
+
+interface DailyData {
+  date: string;
+  workTime: string[];
+}
+
+const Calendar = ({ selectedId }: Props): JSX.Element => {
+  const today = new Date();
+  const year = today.getFullYear();
+  const month = today.getMonth() + 1;
+
+  const { data: scheduleData } = useQuery(
+    ['getMonthly', year, month, selectedId],
+    () => getMonthly(year, month, selectedId),
+    {
+      suspense: true,
+      enabled: selectedId !== 0,
+    },
+  );
+
+  const [selectedDate, setSelectedDate] = useAtom(dateAtom);
+
+  const dateOnClick = (isFixed: boolean, date: string) => {
+    const newObj = { date: date, isFixed: isFixed };
+    setSelectedDate((prev) => newObj);
+  };
+
+  if (!!scheduleData) {
+    return (
+      <>
+        <FlexContainer $direction="row" $justify="space-between">
+          <PrevButton />
+          <Text size="lg" weight="bold">{`${year}  ${month + 1} 월`}</Text>
+          <NextButton />
+        </FlexContainer>
+
+        <MonthBox $wFull>
+          {scheduleData.table.map((weekArray: DailyData[], i) => (
+            <WeekGrid key={`${i}주`}>
+              {weekArray.map((e: DailyData) => (
+                <DayBox
+                  key={e.date}
+                  dateString={e.date}
+                  timeList={e.workTime}
+                  onClick={() => dateOnClick(e.workTime !== null, e.date)}
+                  isSelected={selectedDate.date === e.date}
+                />
+              ))}
+            </WeekGrid>
+          ))}
+        </MonthBox>
+      </>
+    );
+  }
+  return <></>;
```

Comment:
!!를 통해서 boolean으로 만들고 컴포넌트 본체(?)가 if문 내에서 리턴되는게 부자연스러운 것 같아요
Fragment를 먼저 리턴시켜주는게 더 읽기 쉬울 것 같습니다!

```suggestion
  if (!scheduleData) {
    return <></>;
  }
  
  return (
      <>
        <FlexContainer $direction="row" $justify="space-between">
          <PrevButton />
          <Text size="lg" weight="bold">{`${year}  ${month + 1} 월`}</Text>
          <NextButton />
        </FlexContainer>

        <MonthBox $wFull>
          {scheduleData.table.map((weekArray: DailyData[], i) => (
            <WeekGrid key={`${i}주`}>
              {weekArray.map((e: DailyData) => (
                <DayBox
                  key={e.date}
                  dateString={e.date}
                  timeList={e.workTime}
                  onClick={() => dateOnClick(e.workTime !== null, e.date)}
                  isSelected={selectedDate.date === e.date}
                />
              ))}
            </WeekGrid>
          ))}
        </MonthBox>
      </>
    );
```